content/development/clients/clients.en.yaml

```diff
@@ -35,7 +35,8 @@ items:
     key: OpenETC
     name: OpenETC
     link: https://github.com/openetc
-    description: OpenETC is a fast and feature-rich multi-network Ethereum client. For Phoenix upgrade support, please use version 3.0.1 or later. Upgrade considerations, OpenETC is a branch of the deprecated ParityEthereum project. Users can use their existing data directories and do not need to resync for migration to OpenETC from the deprecated ParityEthereum v2.7.2 client. OpenETC is an open source, community maintained branch of the ParityEthereum client. Note this client project is marked as an experimental client due to its infancy. The open development team lacks a known track record of development due to their anonymous nature. Please do your own research if you intend to use this client.
+    warning: OpenETC is a new, experimental community project. For production deployments, please consider a more mature client.
```

Comment:
please see my suggestion here.
https://github.com/ethereumclassic/ethereumclassic.github.io/pull/531#issuecomment-664374247
It should sound not censoring.